src/lib/core/coio.cc

```diff
@@ -238,50 +236,6 @@ coio_connect_timeout(struct ev_io *coio, struct uri *uri, struct sockaddr *addr,
 	tnt_raise(SocketError, sio_socketname(coio->fd), "connection failed");
 }
 
-/**
- * Wait a client connection on a server socket until
- * timedout.
- */
-int
-coio_accept(struct ev_io *coio, struct sockaddr *addr,
```

Comment:
I don't like the idea of removing those functions. I see we are not using them now and the code is not tested at all. But I'd to have coio* as a generic independent library that allows to perform (any) I/O in terms of cooperative concurrency. And there must be, for example, `coio_accept` because it make this library complete.
I agree with UDP (sendto/recvfrom) functions removal though because they are easy to write again and look extraneous to proposed streaming I/O.

Reply:
Makes sense. I resurrected those functions - now this patch set drops only sendto/recvfrom while other functions are modified to take/return raw fd instead of ev_io.